java/client/src/test/java/glide/api/RedisClientTest.java

```diff
@@ -293,6 +299,215 @@ public void exists_returns_long_success() {
         assertEquals(numberExisting, result);
     }
 
+    @SneakyThrows
+    @Test
+    public void expire_returns_success() {
+        // setup
+        String key = "testKey";
+        long seconds = 10L;
+        String[] arguments = new String[] {key, Long.toString(seconds)};
+
+        CompletableFuture<Boolean> testResponse = mock(CompletableFuture.class);
+        when(testResponse.get()).thenReturn(true);
+
+        // match on protobuf request
+        when(commandManager.<Boolean>submitNewCommand(eq(Expire), eq(arguments), any()))
+                .thenReturn(testResponse);
+
+        // exercise
+        CompletableFuture<Boolean> response = service.expire(key, seconds);
+
+        // verify
+        assertNotNull(response);
```

Comment:
why is it different than other tests?
(`assertEquals(testResponse, response);`)

Reply:
Fixed.